### DISCUSSION TITLE
Setting title and canonical for blog list page

### DISCUSSION THREAD

**Discussion by pranay01** [facebook/docusaurus#6217] [Q&A]
We have multiple blog directories hosted under the same site. 

1. How do I set the titles for each of the blog list pages? As of now its taking a generic title. e.g This is a blog https://github.com/SigNoz/signoz.io/tree/main/opentelemetry . The title for https://signoz.io/opentelemetry is just set to be `Blog | SigNoz`.  How do I customise this title?

2. The canonical of the blog list page is getting set randomly. e.g. for  https://signoz.io/opentelemetry canonical is set to https://signoz.io/blog/opentelemetry-nodejs/ . I want the canonical URL to be the same as URL of the page - https://signoz.io/opentelemetry/ in this case.  How do I solve this? It was [suggested](https://github.com/facebook/docusaurus/discussions/5924) that I reswizzle the component, which I did. But still this is not fixed.

Any suggestions would be helpful

**Comment by slorber** [comment]
1: blogTitle should work? https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog#blogTitle

2: you are probably using an older version or you have swizzled some components. We had a similar bug but it should be fixed now.

**Reply by pranay01** [reply]
1. ok - will try setting 
2. We are on beta.14 and we recenly re-swizzled components. Is there a way to check what could be the issues there

**Reply by slorber** [reply]
> We are on beta.14 and we recenly re-swizzled components. Is there a way to check what could be the issues there


Delete all your swizzled components and run the code locally. If the canonical url is still wrong, then open a bug report with a proper repro (like a branch to your main repo). 

I can't tell for sure that you didn't re-introduce weird behavior when swizzling your components, and I can't tell for sure that what I see on your production website is what the code shows 😅 the best is always to remove your own customizations and figure out if it's a docusaurus bug or a customization bug by yourself
